Cavallo.py

```diff
@@ -0,0 +1,77 @@
+from Pezzo import Pezzo
+
+class Cavallo(Pezzo):
+    #implementazione del cavallo
+
+    def __init__(self, colore, posizione=None):
+        super().__init__(colore, posizione, 'Cavallo')
+        self.graphic_rep = '\u2658' if self.colore == 'W' else '\u265E'
+
+    def verifica_mossa(self, destinazione):
+        """ la sintassi è la stessa usata nella funzione verifica_mossa della classe Torre"""
+
+        if super().verifica_mossa(destinazione):  # le condizioni generiche sono verificate
+            #nota la destinazione, calcolo dove il cavallo puo' andare
+            possible_position=self.possibiliPosizioni()
+            if destinazione in possible_position:
+                if not self.scacchiera.get_pezzo([destinazione[0], destinazione[1]]) == None:  # la casella è occupata
```

Comment:
Questo test è sbagliato. Il cavallo può spostarsi in una casella occupata se il pezzo che la occupa è di un altro colore (il caso dello stesso colore è già gestito dai controlli generici.